lib/payloads.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const defaultPayload = {
+  timestamp: undefined
+, line: undefined
+, level: undefined
+, app: undefined
+, env: undefined
+, meta: undefined
+}
+
+const agentPayload = {
+  t: undefined
+, label: undefined
+, line: undefined
+, f: undefined
```

Comment:
`f` is short version for `file` here - so, you can use `file` to look more descriptive

Reply:
`app` and `file` are kinda similar; the only difference is if `file` was given as `/var/log/logdna-agent.log` and `app` wasn't given, in the server-side, `app` gets info from `file` and becomes `logdna-agent.log`

Reply:
I'm wondering whether you can have `basePayload` since most will have the same base - no need to repeat the fields

Reply:
agreed that we can just stop using the shorthand (afaict there is nothing requiring us to use them anymore on the ingestion side)

RE: `basePayload` the only properties these payloads share appear to be `line` and `timestamp` though

Reply:
I hear you on both of these, and I experimented with changing it, but I had a few problems with the ingester having a priority order for `line.f || line.line`.  I'd prefer we leave these as-is for now, just in case we have to debug things when the agent starts using this.  Keeping it apples-to-apples as closely as possible for now (speaking about the payload implementation of the agent).

Reply:
yeah, I think we can continue to tweak it, especially with regards to going with `line` instead of `l`.  The precedence order for `app` versus `f` is tricky in the ingester, therefore it made sense to have these different payloads where the agent does NOT send `app` at all (as opposed to the default payload in the client which has it on every request).

Reply:
With regards to having `basePayload`, I don't think it buys us much. I'd prefer to have explicit definitions for each.  We start to have problems when there's so much sharing going on.